src/main/java/org/craftercms/studio/impl/v1/repository/git/GitContentRepository.java

```diff
@@ -207,7 +207,9 @@ public String writeContent(String siteId, String path, InputStream content) {
                             .replace(REPO_COMMIT_MESSAGE_USERNAME_VAR, username)
                             .replace(REPO_COMMIT_MESSAGE_PATH_VAR, path);
                     commitId = helper.commitFiles(repo, siteId, comment, user, path);
-                    insertProcessedCommitId(siteId, commitId);
+                    if (commitId != null) {
```

Comment:
This is not related to the ticket. Usually, we need a ticket associated with these updates.

Reply:
Added PR https://github.com/craftercms/studio/pull/3600